bin/mqsub_aqua

```diff
@@ -204,6 +204,10 @@ class script_format:
 
     @staticmethod
     def tail(outfile):
+        if args.scratch_data:
+            for dir in args.scratch_data:
+                print("\n#Delete scratch-data from /scratch",file=outfile)
+                print("if [[ -d $MSCRATCH ]]; then rm -rf $MSCRATCH || exit 1; fi",file=outfile)
```

Comment:
Is this safe? If some strange user modifies $MSCRATCH then things could go south.

What if we test that the `pwd -P` of $MSCRATCH is in /scratch/cmr_mqsub first?